descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Edge search srs API",
```

Comment:
Being a spring-based module, i gather that you need to fix maven replacement tokens in ModuleDescriptor [FOLSPRINGB-4](https://issues.folio.org/browse/FOLSPRINGB-4)